lahja/_utils.py

```diff
@@ -0,0 +1,26 @@
+import asyncio
+import pathlib
+import time
+
+
+def wait_for_path_blocking(path: pathlib.Path, timeout: int=30) -> None:
```

Comment:
It's sad there isn't a cross-platform way to do this :cry: 

On linux you have [inotify](http://man7.org/linux/man-pages/man7/inotify.7.html) but that requires [a dependency](https://pypi.org/project/inotify/), it looks like [libfswatch](https://github.com/emcrisostomo/fswatch) abstracts the platform away from you but I can't find any python wrapper.

Reply:
>It's sad there isn't a cross-platform way to do this

Wait, but [`pathlib.Path`](https://docs.python.org/3/library/pathlib.html) should work cross-platform or are we talking about different things?

Reply:
I mean the spinloop to check whether the file exists, `inotify` would let you create a file descriptor which you could `read` in order to block until an event (the file was created) occurred. It would block the thread without eating up CPU time.

Reply:
Ah! Gotcha. Yes, I was quite surprised not to find something like that in the stdlib.